frontend/src/app/contacts/contacts-add/contacts-add.component.html

```diff
@@ -27,12 +27,14 @@ <h3>
     <div class="row" formGroupName="adresse">
         <div class="col-md-12">
             <mat-form-field>
-                <mat-label>Select an option</mat-label>
-                <mat-select matInput placeholder="Land wählen"  [(ngModel)]="contact.adresse.land" name="country">
-                    <mat-option *ngFor="let country of countries" [value]="country.value">
+                <mat-label>Land</mat-label>
+                <!-- Mit mat-select konnte ich ddas Dropdown-Auswahlfeld nicht realisieren. Deshalb native html select -->
+                <select matNativeControl formControlName="land" name="country">
+                    <option value="" selected></option>
+                    <option *ngFor="let country of countries" [value]="country.value">
```

Comment:
warum hat das mit mat-select nicht funktioniert?

Reply:
Ich kann mich jetzt leider nicht mehr an die konkrete Fehlermeldung in der Browser-Konsole erinnern. Irgendwas mit "form-field needs to have a formControl" oder so ähnlich.

Außerdem wurde kein Dropdown angezeigt sondern die Werte "Deutschland" usw. einfach unterienander aufgelistet. Und es konnte keiner dieser Werte ausgewählt werden...

Schwierig, jetzt zu beschreiben.

Reply:
Könntest du das eventuell mit Michelle im pair programming anschauen? falls Michelle keine Zeit hat kann ich auch gerne mit dir anschauen. :)

Reply:
Hab jetzt den auskommentierten Codeschnipsel mit mat-select mit eingefügt.

Wir können das gerne zusammen testen. Ich frag mal Michelle wann sie dafür Zeit hat. Vielleicht Montag vor oder nach dem weekly...